home/src/main/resources/rdf/shapes/everytime/dynamic_api_shapes.n3

```diff
@@ -0,0 +1,730 @@
+@prefix :      <https://vivoweb.org/ontology/vitro-dynamic-api#> .
```

Comment:
Is this file aligned with changes in vitro-dynamic-api.n3? I believe this file might be regenerated by using https://shacl-play.sparna.fr/play/convert, if that is needed

Reply:
It was aligned, but I am sure we will modify ontology in near future and would have update the rules